src/w2fields.js

```diff
@@ -274,7 +274,7 @@
                     };
                     this.options = $.extend(true, {}, defaults, options);
                     options = this.options; // since object is re-created, need to re-assign
-                    if ($(this.el).attr('placeholder') == null) $(this.el).attr('placeholder', options.placeholder || options.format);
+                    if ($(this.el).attr('placeholder') == null) $(this.el).attr('placeholder', options.placeholder == '' ? '' : options.placeholder ? options.placeholder : options.format);
```

Comment:
options.placeholder is no longer used. You need to define placeholder in HTML code
```html
<input id="date" placeholder="">
```
and if you define empty one, then no default placeholder will be visible. I verified in current master, all is good.